app/src/main/java/com/github/se/orator/model/speaking/AnalysisData.kt

```diff
@@ -21,14 +21,6 @@ data class AnalysisData(
       Log.e("AnalysisData", "Invalid transcription: cannot be blank.")
       return false
     }
-    if (fillerWordsCount < 0) {
-      Log.e("AnalysisData", "Invalid fillerWordsCount: cannot be negative.")
-      return false
-    }
-    if (averagePauseDuration < 0.0) {
-      Log.e("AnalysisData", "Invalid averagePauseDuration: cannot be negative.")
-      return false
-    }
     if (talkTimePercentage < 0.0 || talkTimePercentage > 100.0) {
```

Comment:
For the future, I would remove these metrics completely since we did not implement them in the app functionalities.